static/images/platform-icons/Insights.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These icons are available as static resources (`/apps/landing/fonts/Insights.svg`). Check the landing page for the image srouce.